tagz.py

```diff
@@ -221,7 +221,7 @@ def main():
 
         # Identify the latest two tags.
         prev_tags = (git(path,
-                         'for-each-ref refs/tags --sort=-committerdate '
+                         'for-each-ref refs/tags --sort=-refname --sort=-committerdate '
```

Comment:
ah, very good catch. and yeah that's a bit confusing that the last key is the most primary sort key, but good to know that's supported :smile: